resources/assets/icons/email.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="at" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M14.2 10c0 2.3-1.9 4.2-4.2 4.2S5.8 12.3 5.8 10 7.7 5.8 10 5.8s4.2 1.8 4.2 4.2h0z"/><path d="M14.2 10v1.8c0 1.3 1.1 2.4 2.4 2.4s2.4-1.1 2.4-2.4h0V10c0-5-4-9-9-9s-9 4-9 9 4 9 9 9c1.9 0 3.8-.6 5.4-1.8"/></g></svg>
```

Comment:
@ItsANameToo are you happy with me naming this `email.svg` or would you prefer the original name of `at.svg`?